salt/metalk8s/orchestrate/deploy_new_node.sls

```diff
@@ -1,16 +1,19 @@
-{%- if pillar['bootstrap_id'] %}
-{%-   set control_plane_ips = salt.saltutil.runner('mine.get', tgt=pillar['bootstrap_id'], fun='control_plane_ip') %}
-{%- else %}
-{%-   set control_plane_ips = {} %}
-{%- endif %}
-
-{%- if pillar['bootstrap_id'] in control_plane_ips.keys() and control_plane_ips[pillar['bootstrap_id']] %}
-{%-   set control_plane_ip = control_plane_ips[pillar['bootstrap_id']] %}
-{%- else %}
-{%-   set control_plane_ip = 'localhost' %}
-{%- endif %}
+{%- set control_plane_ip = salt.saltutil.runner(
```

Comment:
What's the difference with using `salt['mine.get'](...)`?

Reply:
`salt['mine.get']` doesn't work as we execute it on the master we need to use the runner

Reply:
`salt-run salt.cmd mine.get '*' control_plane_ip` do not return anything but `salt-run mine.get '*' control_plane_ip` does